events/tests/test_organization_api.py

```diff
@@ -1,247 +1,399 @@
 import pytest
 from django_orghierarchy.models import Organization
+from rest_framework import status
 
+from events.api import OrganizationDetailSerializer
 from events.tests.utils import versioned_reverse as reverse
 
 
-@pytest.mark.django_db
-def test_post_no_parent(user, organization, data_source, api_client):
+def organization_id(pk):
+    obj_id = reverse(OrganizationDetailSerializer.view_name, kwargs={"pk": pk})
+    return obj_id
+
+
+def get_organization(api_client, pk):
+    url = reverse(OrganizationDetailSerializer.view_name, kwargs={"pk": pk})
+
+    response = api_client.get(url, format="json")
+    return response
+
+
+def create_organization(api_client, organization_data):
     url = reverse("organization-list")
+
+    response = api_client.post(url, organization_data, format="json")
+    return response
+
+
+def assert_create_organization(api_client, organization_data):
+    response = create_organization(api_client, organization_data)
+
+    assert response.status_code == status.HTTP_201_CREATED
+    return response
+
+
+def delete_organization(api_client, pk):
+    url = reverse(OrganizationDetailSerializer.view_name, kwargs={"pk": pk})
+
+    response = api_client.delete(url)
+    return response
+
+
+def assert_delete_organization(api_client, pk):
+    response = delete_organization(api_client, pk)
+    assert response.status_code == status.HTTP_204_NO_CONTENT
+    return response
+
+
+def update_organization(api_client, pk, organization_data):
+    url = reverse(OrganizationDetailSerializer.view_name, kwargs={"pk": pk})
+
+    response = api_client.put(url, organization_data, format="json")
+    return response
+
+
+def assert_update_organization(api_client, pk, organization_data):
+    response = update_organization(api_client, pk, organization_data)
+    assert response.status_code == status.HTTP_200_OK
+    return response
+
+
+@pytest.mark.django_db
+def test_admin_user_can_see_organization_users(api_client, organization, user):
+    organization.regular_users.add(user)
     api_client.force_authenticate(user)
```

Comment:
There is also `user_api_client` fixture that has the `user` already force authenticated 🙂

Reply:
Great, I didn't know about that. Changed `api_client` to `user_api_client` in all the possible tests in `events/tests/test_organization_api.py` file.

Fixed in https://github.com/City-of-Helsinki/linkedevents/pull/609/commits/e28447789e8f99dad0823b7496a6f3538b8a5484